crew

```diff
@@ -5,9 +5,40 @@ require 'uri'
 require 'digest/sha2'
 require 'json'
 require 'fileutils'
+require 'docopt'
+
+doc = <<DOCOPT
+Chromebrw - Package manager for Chrome OS http://skycocker.github.io/chromebrew/
+
+Usage:
+  #{__FILE__} build [-k|--keep] <name>...
+  #{__FILE__} download <name>...
+  #{__FILE__} help [<command>]
+  #{__FILE__} install [-k|--keep|-s|--build-from-source] <name>...
+  #{__FILE__} remove <name>...
+  #{__FILE__} search [-d|--desc] [<name>...]
+  #{__FILE__} update
+  #{__FILE__} upgrade [-k|--keep|-s|--build-from-source] [<name>...]
+  #{__FILE__} whatprovides <name>...
+
+  -k --keep               Keep extracted files as is.
```

Comment:
It's what crew works right now.  Modifying crew behavior is out of this PR.  Your are welcome to improve it to make it sense.